benchmark/json_parser.h

```diff
@@ -83,31 +84,55 @@ stage2_functype* get_stage2_func(const Architecture architecture) {
   }
 }
 
+jsonparse_functype* get_jsonparse_func(const Architecture architecture) {
```

Comment:
Another note: one day, these functions or something like them should be part of the API surface. People should be able to change architecture if they *want* to.

Reply:
Agreed.